test/saithrift/tests/sail2.py

```diff
@@ -379,3 +379,93 @@ def runTest(self):
             self.client.sai_thrift_set_port_attribute(port2, attr)
             self.client.sai_thrift_set_port_attribute(port3, attr)
             self.client.sai_thrift_set_port_attribute(port4, attr)
+			
+class L2SonicVlanTest(sai_base_test.ThriftInterfaceDataPlane):
+    def runTest(self):
+        print "[access vlan=10]"
```

Comment:
put print as comment using """comment""" to describe the test. Will be easier to create document in the future.

The comment looks like:
Vlan broadcast and known unicast test. Verify the broacast packet reaches all ports in the vlan and unicast packet reach  specific port. 
Steps:
1. remove all ports from vlan
2. create vlan 10
3. add n-1 ports to vlan 10
4. add mac for each port
5. send untagged broadcast packet from port 1, verify all n-1 ports receive the packet except the last port
6. send untagged unicast packet from port 1 to port 2-n. Verify port [2,n-1]th receives the packet and port n does not.
7. clean up.